### ISSUE TITLE
GPU fans at 0 RPM after the FanControl program is deleted 

### ISSUE THREAD

**Issue by chase159** [Rem0o/FanControl.Releases#1421]
I deleted the program from my computer. CPU RPM goes back to normal Basically what I had set in my BIOS. But the GPU fan does not return to its normal settings. The fans do not spin, and they stay at 0 RPM.  

I'm not exactly sure what's going on here and would like some help. 

I have a core I9 10900f With 32 GB of RAM z490 tomahawk motherboard one terabyte of storage and an EVGA FTW3 3080 10 GB

**Comment by Rem0o** [comment]
And if any load comes up on your GPU, does it spins up?  Many GPU will 0 RPM at idle.

**Comment by chase159** [comment]
For some added context The fans on my GPU have always spun at idle And slowly ramp up With higher temperatures. After I installed the fan control. I Let the system detect all my fans And after tweaking some values. On my CPU and GPU, I later deleted the program. Thinking that everything would revert back to its default settings. My CPU definitely goes back to its default settings. But now after I've done a little bit more testing My GPU fans only spin up at high load. Everything else below 70 degrees Celsius Reverts to 0 RPMs Which is not Usual behaviour for my GPU. I'm just rather confused why my CPU went back to its normal fan curve but my GPU did not.After Deleting the program. I'm not sure if you'll have the fix but I thought I might ask.

**Comment by Rem0o** [comment]
FanControl doesn't alter anything GPU wise. It's not like it changes the bios or anything in the GPU. Technically, as soon as you reboot, whatever FanControl applied gets wiped. Maybe you had a fancurve in afterburner or something before?

**Comment by chase159** [comment]
No everything is pretty much stock after I built my computer. I haven't installed any Fan controlling software besides this one. I'm unsure if reinstalling Windows would fix the problem or if reinstalling my graphics driver would be the fix. I don't want to assume it's the fan control application that's the problem, mainly because when I delete it and restart my PC everything works fine, but my GPU fan curve / RPM Doesn't go back to normal.  I just find it weird that After I installed fan control. And after deleting it I now have this issue. Sorry if I sound like I'm repeating myself. But I just don't understand how Everything but my GPU fans are fine.

**Comment by chase159** [comment]
Hi there. Here's an update on my current situation. My problem is resolved. My solution 

Uninstall the program completely 

Shut down the PC 

Unplug my power cord  

Cycle the Power button a few times 

Change my GPU From normal to Overclock 

Boot into BIOS. Then disable fast boot 

After that I booted into windows and it worked just fine. 

My GPU is now working at its default settings. I'm not exactly sure how it stopped functioning correctly After I uninstalled fan control. But somehow disabling fast boot fixed the problem. My system will boot slower now that I have turned off fast boot but that doesn't matter to me. I hope this can be helpful for anyone having a similar issue in the future.
